modules/ROOT/pages/java-support.adoc

```diff
@@ -65,7 +65,11 @@ To enhance application performance, see:
 
 Java 17 manages memory differently than it did in earlier versions like Java 8. Java 17 reduces request latency and executes work more predictably. To achieve this, it adopts a different memory management approach. Although highly dependent on each application, in some cases, Java 17 preferentially commits to heap memory.
 
-Because of this change in Java 17 memory management, to ensure optimal operation and effectively leverage these improvements, you must conduct xref:mule-runtime::tuning-test-validations.adoc[performance testing] and xref:mule-runtime::tuning-monitoring.adoc[monitoring] for your most common workloads.
+[NOTE]
```

Comment:
The note is not including the bullets. Is this on purpose?

Otherwise, approved.